SymSettings.cfg

```diff
@@ -1,28 +1,48 @@
+<<<<<<< HEAD
+set SEED -1                    # What value should the random seed be? If seed <= 0, then it is randomly re-chosen.
+set DATA_INT 1                 # How frequently, in updates, should data print?
+set MUTATION_SIZE 0            # Standard deviation of the distribution to mutate by
+set HORIZ_MUTATION_SIZE 0      # Standard deviation of the distribution to mutate by for horizontal transmission, if -1 MUTATION_SIZE used
+set MUTATION_RATE 0            # Value 0 to 1 of probability of mutation
+set HORIZ_MUTATION_RATE 0      # Value 0 to 1 of probability of mutation for horizontal transmission, if -1 MUTATION_RATE used
+=======
 ### MAIN ###
 # Global Settings
 
 set SEED 10                    # What value should the random seed be? If seed <= 0, then it is randomly re-chosen.
 set COMPETITION_MODE 0         # Should a competition between two types be conducted? (Which is specified in the code)
-set DATA_INT 100               # How frequently, in updates, should data print?
```

Comment:
Make sure not to change the default sym Settings, best to rename the ones you are currently using and then run symbulation to generate a default one